trashcli/restore.py

```diff
@@ -117,21 +117,30 @@ class Command:
 
 def parse_args(sys_argv, curdir):
     import argparse
+    try:
+        import shtab
+    except ImportError:
+        from . import _shtab as shtab
+
     parser = argparse.ArgumentParser(
+        'trash-restore',
         description='Restores from trash chosen file',
         formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    shtab.add_argument_to(parser, preamble=PREAMBLE)
```

Comment:
Also this logic is duplicated, can you create a single entry point (a function) that does the followings?
1) it takes in input a ArgumentParser
2) if available imports shtab
3) if shtab is available calls the `shtab.add_argument_toi(parse, preamble=PREAMBLE)`
4) if shtab is not available should make that the .complete Action becomes a no op
5) if shtab is not available add a '--print-completion' option that tell the user to install shtab

The function should be placed in `trashcli/shell_completion.py`